samples/bluetooth/iso_broadcast_benchmark/src/broadcaster.c

```diff
@@ -411,8 +410,7 @@ static int create_big(struct bt_le_ext_adv **adv, struct bt_iso_big **big)
 	/* Set periodic advertising parameters */
 	err = bt_le_per_adv_set_param(*adv, BT_LE_PER_ADV_DEFAULT);
 	if (err != 0) {
-		LOG_ERR("Failed to set periodic advertising parameters (err %d)",
-			err);
+		LOG_ERR("Failed to set periodic advertising parameters (err %d)", err);
```

Comment:
The changes in this file seems unnecessary.

Reply:
Hmm, this seems to be the case for several files. Are you using some kind of auto-format that is modifying lines of code, just to make them a single line? 

If so, please turn that off, and revert those changes.

Reply:
These changes stem from this comment: https://github.com/zephyrproject-rtos/zephyr/pull/51897#discussion_r1011809309
I don't think it's reasonable to ask him to format all of those changes manually. 

He's running clang-format on his changes with the zephyr rules, if you don't like the formatting, please update the rules or provide an automated tool to do it.

This might seem like a bad-faith argument from me, since clang-format has some limitations, but I really think those nitpicky formatting comments (by you, me or anyone else) shouldn't be necessary and should be encoded in the CI checks, and if they can't, we should refrain from commenting/requesting changes.

I haven't used it a lot, but maybe coccinelle can do this sort of thing? pinging @carlescufi if he knows, since that's a recurring issue in PRs.

Reply:
I agree! We really need to just get everything clang-formatted. Formatting issues are consistently a big impediment to merging PRs in Zephyr. That's a waste of everyone's time in my opinion.

Reply:
> He's running clang-format on his changes with the zephyr rules, if you don't like the formatting, please update the rules or provide an automated tool to do it.

If that was the case, then that would be fine, but this case (and many more) is not changed by his PR, except to remove this line brake. 

This PR is disruptive enough already (I'll have to rebase dozens of commits after this already) without modifying lines just for the sake of modifying it. 

Besides this specific line also makes the line longer than 80 characters, which is the recommended maximum. 

To me it seems that he has incorrectly configured his clang-format, and neither of the following should occur in this PR:
1) We should not remove newlines the same of removing newlines if the code already follows the code standards (which it did). 
2) We should not automatically make any line exceed the 80 character recommended maximum, unless it significantly improved readability. 

IMO this PR breaks our current coding standards.